### PR TITLE
Add media viewer

### PR DESCRIPTION
**Pull request by linusnorton**

### CODE REVIEW

File: src/frontend/angular.json [hunk 1]
```diff
@@ -34,11 +34,17 @@
...
+                "glob": "**/*",
+                "input": "node_modules/@hmcts/media-viewer/assets",
+                "output": "/assets"
               }
             ],
             "styles": [
               "node_modules/govuk-frontend/all.scss",
-              "node_modules/@hmcts/dg-docassembly-webcomponent/assets/aui-styles.scss"
+              "node_modules/@hmcts/dg-docassembly-webcomponent/assets/aui-styles.scss",
+              "node_modules/@hmcts/media-viewer/assets/aui-styles.scss"
```
Comment (benouaer) [inline src/frontend/angular.json]: that's probably the same file, do we need to add it again

File: src/frontend/src/app/media-viewer-wrapper/media-viewer-wrapper.component.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { ActionEvents } from "@hmcts/media-viewer";
+
+@Component({
+  selector: 'app-media-viewer-wrapper',
+  templateUrl: './media-viewer-wrapper.component.html',
+  styleUrls: ['./media-viewer-wrapper.component.scss']
+})
+export class MediaViewerWrapperComponent {
+
+  documentTypeToShow = 'nonDM_PDF';
```
Comment (benouaer) [inline src/frontend/src/app/media-viewer-wrapper/media-viewer-wrapper.component.ts]: I think this should be set to pdf

Comment (linusnorton) [reply src/frontend/src/app/media-viewer-wrapper/media-viewer-wrapper.component.ts]: copy paste, will see what it does

File: src/frontend/angular.json (hunk 1 above)
Comment (linusnorton) [reply src/frontend/angular.json]: no idea

File: src/frontend/src/app/media-viewer-wrapper/media-viewer-wrapper.component.ts (hunk 2 above)
Comment (benouaer) [reply src/frontend/src/app/media-viewer-wrapper/media-viewer-wrapper.component.ts]: it's used to select the tab, and our tab values are pdf, image, and unsupported, so default is probably pdf

**Pull request merged**